update-and-commit-translations.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Script to pull transifex translation files, and commit them to git if there are any changes.
+
+tx pull -f --mode=reviewed
+
+if [[ -z $(git diff) ]]; then
```

Comment:
While the git diff might be working today, the behaviour of the command can change drastically from one version to another. 

You could use `git diff-index --quiet HEAD -- ` for example, as it's considered a 'plumbing' command and should not break. 

Everything else should work just fine.